examples/ChatQnA/deploy/gaudi.md

```diff
@@ -5,10 +5,9 @@ example with OPEA comps to deploy using vLLM or TGI service. There are several
 slice-n-dice ways to enable RAG with vectordb and LLM models, but here we will
 be covering one option of doing it for convenience : we will be showcasing how
 to build an e2e chatQnA with Redis VectorDB and neural-chat-7b-v3-3 model,
-deployed on Intel® Tiber™ Developer Cloud (ITDC). For more information on how to setup ITDC instance to proceed,
-Please follow the instructions here (*** getting started section***). If you do
-not have an ITDC instance or the hardware is not supported in the ITDC yet, you can still run this on-prem. To run this on-prem, make sure that all the
-(***system level requriements***)  are addressed such as docker versions, driver version etc.
+deployed on Intel® Tiber™ Developer Cloud (ITDC). To quickly learn about OPEA in just 5 minutes and set up the required hardware and software, please follow the instructions in the
+[Getting Started](https://opea-project.github.io/latest/getting-started/README.html) section. If you do

```

Comment:
This links to the latest (main) documentation.  To keep the link within the released documentation you should use relative links to the file name to link to the document when that doc is within the same repo.  So, in this case, use `[Getting Started](/getting_started/README.md)`.  If the file is in a another OPEA repo, link to the source file (.md or ,,rst) with an URL to that file in the github.com repo.  The Sphinx processing converts references to .md files to the generate HTML file.